crates/graphql_artifact_generation/src/generate_artifacts.rs

```diff
@@ -503,6 +501,7 @@ derive_display!(ReaderAst);
 pub(crate) struct NormalizationAstText(pub String);
 derive_display!(NormalizationAstText);
 
+#[allow(dead_code)]
```

Comment:
You can go ahead and remove this struct entirely! Thanks for finding this!

Reply:
Removed.